test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9613,6 +9613,22 @@ def create_hook(self, name, attrs):
         self.manager(MGR_CMD_SET, HOOK, attrs, id=name)
         return True
 
+    def delete_hook(self, name):
+        """
+        Helper function to delete a hook by name.
+
+        :param name: The name of the hook to create
+        :type name: str
+        :returns: False if hook does not exist
+        :raises: PbsManagerError, otherwise return True.
+        """
+        hooks = self.status(HOOK, level=logging.DEBUG)
+        for hook in hooks:
+            if hook['id'] == name:
+                self.logger.info("Removing hook:%s" % name)
+                self.manager(MGR_CMD_DELETE, HOOK, id=[name])
```

Comment:
Thanks for adding this helper function!
Just a minor comment - the name does not need to be a list, you can just pass in a string.